huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go

```diff
@@ -3,26 +3,32 @@ package cce
 import (
 	"context"
 	"fmt"
+	"log"
 	"strings"
 	"time"
 
-	"github.com/chnsz/golangsdk"
-	"github.com/chnsz/golangsdk/openstack/aom/v1/icagents"
-	"github.com/chnsz/golangsdk/openstack/cce/v3/clusters"
-	"github.com/chnsz/golangsdk/openstack/cce/v3/nodes"
-	"github.com/chnsz/golangsdk/openstack/common/tags"
 	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/aom/v1/icagents"
+	"github.com/chnsz/golangsdk/openstack/cce/v3/clusters"
+	"github.com/chnsz/golangsdk/openstack/cce/v3/nodes"
+	"github.com/chnsz/golangsdk/openstack/common/tags"
+
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
 )
 
+// Deprecated function name, used only for G42 and OCB references.
+func ResourceCCEClusterV3() *schema.Resource {
```

Comment:
we can modify the code in G42

Reply:
The description of the deprecated function has been updated.